python/mp/rosary.py

```diff
@@ -57,9 +87,29 @@ def __init__(self, ip="127.0.0.1", port=5005, dispatcher=None, name="rosary"):
         self.dispatcher = dispatcher
         # Available knobs to turn
         self.knobs = {}
+        self.updater_list = []
 
+        self.osc_client = udp_client.UDPClient(self.osc_ip, self.osc_port)
+
+        # create the three classes of LED "beads"
         for i in range(self.BEAD_COUNT):
             self.beads.append(Bead(i))
+        self.updater_list.append(Updater(name='rosary',
+                                         bead_list=self.beads,
+                                         osc_client=self.osc_client))
+
+        for i in range(self.BASE_COUNT):
+            self.bases.append(Bead(i))
+        self.updater_list.append(Updater(name='base',
+                                         bead_list=self.bases,
+                                         osc_client=self.osc_client))
+
+        for i in range(self.CROSS_LED_COUNT):
+            self.cross.append(Bead(i))
+        self.updater_list.append(Updater(name='cross',
+                                         bead_list=self.cross,
+                                         osc_client=self.osc_client))
+
```

Comment:
To put it another way, here are some examples:

# run 3phase_sine_wave on the cross
oscsend localhost 5006 /rosary/add_effect/name/3phase_sine_wave/bead_set/cross
# run throb on the bases
oscsend localhost 5006 /rosary/add_effect/name/throb/bead_set/bases
# run bounce on the rosary
oscsend localhost 5006 /rosary/add_effect/name/bounce/bead_set/rosary